src/s1a_portapotty.tw2

```diff
@@ -88,61 +66,66 @@ I reached forward, snatched it up, and recognized its familiar heft immediately.
 
 I gave it a good flick, and the room sprang into detail around me.
```

Comment:
I could see that, yeah. I think it could help to offset the action by putting it on a new line because immediately after clicking it, the player's going to see a new scene image for the first time (the illuminated portapotty). Seems a little more dramatic to me this way, whereas when "flick" is within a sentence, it doesn't really imply to me that "hey, this is going to move the scene along." What do you think?

Reply:
my main thought here is that the current flow came across as seeming either out of order, or slightly unclear. i.e., the sentence describes flicking the lighter, and the room having already sprung into detail around the player. so when playing through, I wasn't sure if the `* click *` interaction represented the scene rewound to before flicking the lighter & the room having sprung into detail, or represented doing something else in the now-lit room (or that some other object in the room began to click of its own accord, which was what I ultimately thought was happening).

a way to clarify might be to have the sentence appear first as:
`I gave it a good flick—`

with all or part of that as a link, and then have that link transition the player to the next passage, finishing the action:
`—and the room sprang into detail around me.`

before continuing with the description of the lit room.

or maybe the lighter sparks at first, too short to get any detail about the room, and then the `* click *` is the player trying again?

Reply:
Hmm. I can see why that could be confusing if you're looking at the text, including the actions, as strictly linear. 

I haven't been thinking of it that way, but I'm not sure exactly how to explain how I see it. I guess in my view, the `* click *` is an example of the player, like, _agreeing_ to what the game has laid out so far, and they're consenting that that is the action taken? I admit this is probably influenced by the fact that I recently discovered [Homestuck](https://www.homestuck.com) and I've been thinking a lot about the relationship between narrator and "player" there.

@catdad666, I'd love to hear your thoughts here too!

Reply:
This is definitely something we can address down the line, and I don't wanna hold up image integration, so I'm gonna make a new issue for this: https://github.com/whymog/efjm/issues/11